.travis.yml

```diff
@@ -0,0 +1,7 @@
+language: ruby
```

Comment:
I believe this is correctly running the test since the default `script` property is `rake` which runs the tests.

What do you think about making it explicit though? Otherwise if you're not familiar with Travis you need to go check the docs (like I just did 😄) to understand what's going on.

Also, after #1 is merged, it might be a good idea to add rubocop to the CI checks as well.